package.json

```diff
@@ -14,11 +14,14 @@
     "test": "yarn unit-test && yarn integration-test",
     "create-views": "knex seed:run --env views",
     "migrate-app": "knex migrate:latest --env app && node ./delete-views && yarn create-views",
-    "seed-app": "node ./delete-data.js && knex seed:run --env app",
+    "seed-dev": "knex seed:run --env dev",
+    "seed-ref": "knex seed:run --env app",
+    "delete-dev-data": "node ./delete-data.js dev",
+    "delete-ref-data": "node ./delete-data.js ref",
     "rollback-app": "knex migrate:rollback --env app",
     "migrate-stg": "knex migrate:latest --env staging",
     "rollback-stg": "knex migrate:rollback --env staging",
-    "reset-db": "yarn migrate-stg && yarn migrate-app && yarn seed-app"
+    "reset-db": "yarn migrate-stg && yarn migrate-app && yarn delete-dev-data && yarn delete-ref-data && yarn seed-ref && yarn seed-dev"
```

Comment:
Comment from Tom from previous PR:

"Might want to take seed-dev out of this - that way for integration we can run reset-db and seed-dev separately, but all other environments can run this command and get only the ref data.

The Jenkins migration job for integration containing reset-db would need updated to add seed-dev as well."

Reply:
Update to this @tswann : Any environment other than integration will need to just run the migrations. 

To update the ref data, we need to delete the old ref data. 
To delete the old ref data, we need to delete data from all non-ref tables.
This will wipe any migrated real data.

It looks like updating the ref data on later environments will require a full reset of the DB, including migrating over the real data again. Does that seem correct?

Reply:
Actually, we could probably drop and re-add the foreign key constraint and drop the ref data that way.......

Reply:
Actually, actually, that's probably a bad idea